### ISSUE TITLE
Adjust 'open in new window' link text

### ISSUE THREAD

**Issue by ahamelers** [datadryad/dryad-product-roadmap#3790]
- Curators have complained that when copying external links the text "(opens in new window)" can be included
- Additionally, WCAG requirements state that links that open in a new window should have some kind of visual indication for all users—not just hidden text for screen reader users

Adjust the display of links that open in a new window so that a visual element is included, and not copied (like we have for links to PDFs)
